src/components/ItaliaTheme/View/Commons/RichText.jsx

```diff
@@ -15,12 +15,14 @@ const RichText = ({
   children,
   serif = true,
   lighthouseId = '',
+  context = null,
```

Comment:
Mettere qui il default a `null` è un po' "a vuoto", ci sta che abbia un default ma lo metterei nel punto più interno. In altri termini qui toglierei il default e lo metterei in `RenderBlocks`. Ti dirò di più, metterei `{}` come default, è un po' più safe dato che i blocchi potenzialmente si aspettano di averlo sempre un contesto e almeno qualsiasi codice che sia `properties.qualcosa` dentro il blocco non si spacca.